lib/tasks/account.rake

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+require "#{Rails.root}/lib/task_helpers/account_helper"
+require 'csv'
+include AccountHelper
+
+# To use: rake account:generate["your_file_path_here"]
+namespace :account do
+  task :generate, [:file_path] => [:environment] do |_t, args|
+    file = CSV.read(args[:file_path], headers: true, col_sep: "\t", quote_char: "\x00")
+    file.each do |user|
+      if user["sixplus2"].to_s.present?
+        if generate_uc_account(user)
+          puts "Account generated for #{user['first_name']} #{user['last_name']} (#{user['email']})"
+        else
+          puts "Account could not be generated. A user with email #{user['email']} may already exist."
+        end
+      else
+        puts "Usage:   rake account:generate['sixplus2','Firstname','Lastname'] (no spaces between the parameters)"
```

Comment:
Is this usage still valid, or does it always read from file now?

Reply:
It is not! Good call. I'll change that